### DISCUSSION TITLE
adonis on  rarefied and non-rarefied data using Bray–Curtis?

### DISCUSSION THREAD

**Discussion by marwa38** [vegandevs/vegan#495] [Q&A]
Hello 

I am running adonis() permanova test on rarefied and non-rarefied data in phyloseq object. 
But I didn't get significant statistical difference 
I was wondering if beta diversity metrics could be ran on other format of data other than rarefied or non-rarefied?
I am mainly working on PCoA ordination using Bray-Curtis dissimilarity 

Many thanks

**Comment by gavinsimpson** [comment]
I'm sorry, I don't even understand what you are asking.

`adonis()` isn't a beta diversity metric (were you thinking of PERMDISP? In which case see `?betadisper`)

How were you comparing the rarefied and unrarefied data?

(I'm going to migrate this to a discussion to clean up the issue tracker - this isn't an issue in {vegan} from what little info you have provided.)

**Comment by marwa38** [comment]
I was asking here about the way of transformation or scaling to be used before PCoA plotting Bray-Curtis distances. Data without any scaling or transformation, rarefied data, log-transformed data, etc.
